transport/cluster_integration_test.go

```diff
@@ -0,0 +1,93 @@
+//go:build integration
+
+package transport
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"scylla-go-driver/frame"
+	. "scylla-go-driver/frame/response"
+)
+
+const awaitingChanges = 100 * time.Millisecond
+
+func compareNodes(c *Cluster, addr string, expected *Node) error {
+	m := c.Peers()
+	got, ok := m[addr]
+	switch {
+	case !ok:
+		return fmt.Errorf("couldn't find node: %s in cluster's nodes", addr)
```

Comment:
Maybe also log here known hosts from cluster if there won't be too many of them...

Reply:
Do you mean knownHosts as the list of hosts given us by the user or all the nodes in the cluster?

Reply:
I mean it's just a question from my side, would you think that any of your two proposals help debug the potential error?

Reply:
I don't mind adding it.